tuned_lens/scripts/ingredients.py

```diff
@@ -284,7 +294,10 @@ def shard_model(
         elif self.cpu_offload:
             raise ValueError("CPU offload requires FSDP.")
         else:
-            model.to(self.device)
+            # We don't need to move the model to the appropriate device because we
```

Comment:
This comment assumes a lot about how and when this method is called. The fact that the method does not send the model to the device automatically and assumes the model is pre-situated on the appropriate cuda device should be documented in the doc string.

Reply:
It's actually not clear that we need to say anything about this. The name of the method is just `shard_model`. You'd kind of think that this should only shard the model for FSDP if needed, otherwise do nothing.